src/status_im/components/drawer/view.cljs

```diff
@@ -49,6 +50,8 @@
 (defn navigate-to-accounts []
   (close-drawer!)
   (save-profile!)
+  ;; TODO(rasom): probably not the best place for this call
+  (protocol/stop-whisper!)
```

Comment:
Can you write short comment on why we stop whisper when navigating to accounts? Similar to what is in PR description

Reply:
> Also applies to switching accounts, because accounts may be using different networks.
that's it actually. We don't know which network will be used on the next log in.

Reply:
Sorry I meant put that in code just under the TODO, if I come across that in a few months I have no idea why it is there